src/components/dashboard/IQButton.tsx

```diff
@@ -0,0 +1,73 @@
+import { Box, Button, Tooltip, Spinner } from '@chakra-ui/react'
+import React, { useEffect, useState } from 'react'
+import { addToken } from '@/utils/add-new-token'
+import Image from 'next/image'
+import * as Humanize from 'humanize-plus'
+
+export const IQButton = () => {
+  const [price, setPrice] = useState(0)
+  const [isLoading, setIsLoading] = useState(true)
+  useEffect(() => {
```

Comment:
Instead of calling here, I think there's a ``fetchPrices`` function we can make use of.